docs/en/sass-setup.md

```diff
@@ -0,0 +1,67 @@
+# SASS Configuration
+
+The sass is built to be configurable per project, we do this by having a default properties map which we refer to as the "theme", its located here: `sass\variables\_theme.scss`. 
+
+```
+$default-theme-properties: (
+  // Colors
+  primaryColor: #999999,
+  secondaryColor: #000000,
+  tertiaryColor: #cccccc,
+  backgroundColorLight: #ffffff,
+  backgroundColorDark: #333333,
+  textColorLight: #ffffff,
+  textColorDark: #333333,
+
+  // Fonts
+  fontHeader: "Montserrat",
+  fontText: "Roboto",
+  fontIcon: "Font Awesome 5 Pro",
```

Comment:
Should we use the free library here as an example?

Reply:
Yes, thats a good idea!